documentation/gapc on windows.txt

```diff
@@ -0,0 +1,23 @@
+How to compile gapc on Windows 10:
+
+1) Install Linux Sub-System
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) Open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux

```

Comment:
You should try to be consistent with your capitalization, eg. 1.1) *g*_o vs. 1.2) *O*_pen